torch/_logging/_internal.py

```diff
@@ -733,7 +734,11 @@ def format(self, record):
         if (trace_id := torch._guards.CompileContext.current_trace_id()) is not None:
             record.traceid = f" [{trace_id}]"
 
-        prefix = f"{record.rankprefix}[{record.asctime}]{record.traceid} {record.name}: [{record.levelname}]"
+        prefix = (
+            f"{record.rankprefix}{record.levelname[0]}{record.asctime}.{int(record.msecs*1000):06d} {record.process} "
```

Comment:
I think "DEBUG" is "V" and not "D" in fbcode logs. I didn't bother with this in the supervisor code because it doesn't issue debug level logs anyway and it was using %-formatting where it was hard to turn the D into a V anyway. Here we do use debug logs and it is easy to fix.